src/main/java/gov/nih/nci/evs/api/controller/ConceptController.java

```diff
@@ -2140,6 +2142,90 @@ public class ConceptController extends BaseController {
     }
   }
 
+  /**
+   * Returns the codes for a given terminology.
+   *
+   * @param terminology the terminology
+   * @return the terminology codes
+   * @throws Exception the exception
+   */
+  @Operation(summary = "Get codes for the specified terminology")
+  @ApiResponses({
+    @ApiResponse(
+        responseCode = "200",
+        description = "Successfully retrieved the requested information",
+        content = @Content(schema = @Schema(implementation = ArrayList.class))),
+    @ApiResponse(
```

Comment:
Unfortunately you can't use ArrayList.class and have this work properly.  you want to do something like this (this is basically how you say an array/list of strings):

@Content(array = @ArraySchema(
                            schema = @Schema(implementation = String.class)))